src/Compilers/CSharp/Portable/LanguageVersion.cs

```diff
@@ -57,14 +57,25 @@ public enum LanguageVersion
         /// </list> 
         /// </remarks> 
         CSharp6 = 6,
+
         /// <summary>
         /// C# language version 7.
         /// </summary>
         CSharp7 = 7,
+
+        /// <summary>
+        /// The latest version of the language supported.
+        /// </summary>
+        Latest = int.MaxValue,
```

Comment:
@AdamSpeight2008 You should submit a feature request for that. This PR is to introduce a latest version, which is not necessarily the default.

---
In reply to: [72680619](https://github.com/dotnet/roslyn/pull/12796#discussion_r72680619) [](ancestors = 72680619)